src/cluster/replication.h

```diff
@@ -87,8 +93,9 @@ class FeedSlaveThread {
   redis::Request req_;
   std::atomic<rocksdb::SequenceNumber> ack_seq_ = 0;
 
-  static const size_t kMaxDelayUpdates = 16;
-  static const size_t kMaxDelayBytes = 16 * 1024;
+  // Configurable delay limits
+  size_t max_delay_bytes_ = 16 * 1024;
+  size_t max_delay_updates_ = 16;
```

Comment:
```suggestion
  size_t max_delay_bytes_;
  size_t max_delay_updates_;
```

Default value seems not so useful here.